scripts/build_bsp.sh

```diff
@@ -24,7 +24,7 @@ bsp_version=$3
     if [ ! -d trusted-firmware-a ]; then
         cd ${topdir}/build/${build}/bsp_sources
         log ">> atf: not found. cloning .."
-        atf_srcrev=($(read_bsp_config ${bsp_version} atf_srcrev))
+        atf_srcrev="2fcd408bb3a6756767a43c073c597cef06e7f2d5"
```

Comment:
Not needed. This comes from the config file. And looks like it hasn't updated from 09.00 RC5 to 09.0 RC6.